src/Umbraco.Infrastructure/PropertyEditors/MediaPicker3PropertyEditor.cs

```diff
@@ -185,76 +188,77 @@ internal static IEnumerable<MediaWithCropsDto> Deserialize(IJsonSerializer jsonS
-                    {
-                        result.Add(dto);
-                    }
-
-                    continue;
-                }
-
-                var temporaryLocationString = temporaryLocation.Value<string>();
-                if (temporaryLocationString is null)
-                {
-                    continue;
-                }
-
-                GuidUdi? startNodeGuid = mediaPicker3Configuration.StartNodeId as GuidUdi ?? null;
-                JToken? mediaTypeAlias = dto.GetValue("mediaTypeAlias");
-                IMedia mediaFile = _temporaryMediaService.Save(temporaryLocationString, startNodeGuid?.Guid, mediaTypeAlias?.Value<string>());
-                MediaWithCropsDto? mediaDto = _jsonSerializer.Deserialize<MediaWithCropsDto>(dto.ToString());
-                if (mediaDto is null)
-                {
-                    continue;
-                }
-
-                mediaDto.MediaKey = mediaFile.GetUdi().Guid;
-                result.Add(JObject.Parse(_jsonSerializer.Serialize(mediaDto)));
-            }
-
-            return result;
-        }
+        // private JArray PersistTempMedia(JArray jArray, MediaPicker3Configuration mediaPicker3Configuration)
+        // {
+        //     var result = new JArray();
+        //     foreach (JObject? dto in jArray.Values<JObject>())
+        //     {
+        //         if (dto is null)
+        //         {
+        //             continue;
+        //         }
+        //
+        //         if (!dto.TryGetValue("tmpLocation", out JToken? temporaryLocation))
+        //         {
+        //             // If it does not have a temporary path, it can be an already saved image or not-yet uploaded temp-image, check for media-key
+        //             if (dto.TryGetValue("mediaKey", out _))
+        //             {
+        //                 result.Add(dto);
+        //             }
+        //
+        //             continue;
+        //         }
+        //
+        //         var temporaryLocationString = temporaryLocation.Value<string>();
+        //         if (temporaryLocationString is null)
+        //         {
+        //             continue;
+        //         }
+        //
+        //         GuidUdi? startNodeGuid = mediaPicker3Configuration.StartNodeId as GuidUdi ?? null;
+        //         JToken? mediaTypeAlias = dto.GetValue("mediaTypeAlias");
+        //         IMedia mediaFile = _temporaryMediaService.Save(temporaryLocationString, startNodeGuid?.Guid, mediaTypeAlias?.Value<string>());
+        //         MediaWithCropsDto? mediaDto = _jsonSerializer.Deserialize<MediaWithCropsDto>(dto.ToString());
+        //         if (mediaDto is null)
+        //         {
+        //             continue;
+        //         }
+        //
+        //         mediaDto.MediaKey = mediaFile.GetUdi().Guid;
+        //         result.Add(JObject.Parse(_jsonSerializer.Serialize(mediaDto)));
+        //     }
+        //
+        //     return result;
+        // }
```

Comment:
Not sure why this commented code is here, should probably be removed, 
or if useful, have some kind of comment about why its there 😄

Reply:
There's a reference to this commented out code somewhere in the code above; it's meant to be left behind as inspiration for when we can implement support for temporary file uploads again.

![image](https://user-images.githubusercontent.com/7405322/223377948-03554fdf-313e-4205-b023-cb3d29742815.png)

Reply:
Ah i missed that last part, great 🙈